src/actions/core/checkoutManagement/eCommerceB2CFunctions.ts

```diff
@@ -44,16 +44,22 @@ export interface ShoppingCart {
  * @params checkoutInput - Checkout object containing items to be purchased and billing information.
  * @params uuid - Unique uuid for this transaction to prevent duplicate transactions from taking place. Gets included into the "x-acme-request-uuid" header key
  * @params options - option bag parameter.
+ * @params browserIpAddress - IP address of the requesting browser for the checkout action. 
  */
 export async function performCheckout(
 	checkoutInput: CheckoutInputObject,
 	uuid: string,
 	options: { throwRaw?: boolean } = {},
+	browserIpAddress?: string,
 ) {
 	// Destructure optional parameters
 	const { throwRaw = false } = options;
 
-	const additionalHeaders = { "x-acme-request-uuid": uuid };
+	const additionalHeaders = { 
+		"x-acme-request-uuid": uuid, 
+		"x-acme-browser-ip": browserIpAddress || await getUserIpAddress(),
+	};
+
```

Comment:
Similar to how we add the `x-acme-request-uuid` header and pass that to `performRequest` I thought it made sense to do that for `x-acme-browser-ip` as well. That way, it's only passed for the B2C Checkout endpoints, and we fallback to `getUserIpAddress()` just for now

Reply:
Makes sense to me!